src/core/border_router/routing_manager.hpp

```diff
@@ -1016,6 +1017,8 @@ class RoutingManager : public InstanceLocator
     public:
         explicit OmrPrefixManager(Instance &aInstance);
 
+        static constexpr RoutePreference kPdRoutePreference = RoutePreference::kRoutePreferenceMedium;
```

Comment:
Suggest moving this to `PdPrefixManager` as well?

Reply:
I would prefer to keep it here, as the preference is set and published by this class.

Reply:
Makes sense, moved to PdPrefixManager.